src/renderer/pages/project/components/chat/bot_message.vue

```
@@ -4,17 +4,42 @@
       <Bot :size="16" class="text-blue-600" />
     </div>
     <div class="flex-1">
-      <div class="bg-gray-100 text-gray-800 p-3 rounded-lg inline-block max-w-md">
-        <p class="text-sm">{{ message }}</p>
-      </div>
+      <div
+        class="bg-gray-100 text-gray-800 p-3 rounded-lg block max-w-md text-sm break-words whitespace-pre-wrap chat-markdown"
+        v-html="safeHtml"
+        />
```

ESLint

[error] Delete `··`

(prettier/prettier)